src/app/api/crawl/crawler.ts

```diff
@@ -1,5 +1,5 @@
-import cheerio from 'cheerio';
```

Comment:
your linter shouldnt be making changes for this PR

Reply:
Sorting now.